tests/ops/fixtures/saas/datadog_fixtures.py

```diff
@@ -95,3 +95,70 @@ def datadog_dataset_config(
     )
     yield dataset
     dataset.delete(db=db)
+
+
+@pytest.fixture(scope='session')
+def generate_test_data_on_datadog(*args, **kwargs):
+    import requests
+    import json
+
+    url = "https://http-intake.logs.datadoghq.com/api/v2/logs"
+
+    payload = json.dumps([
```

Comment:
We just need a single entry, no need to create multiple.

Reply:
Initially we were entering multiple entries so that system could also check pagination. With just one entry, system won't be able to test pagination on this.